.flowconfig

```diff
@@ -1,11 +1,13 @@
 [ignore]
 .*/lib/.*
+.*/node_modules/jsx-control-statements/jsx-control-statements.flow.js
 .*/root/static/build/.*
 .*/t/.*
 
 [include]
 
 [libs]
+./node_modules/jsx-control-statements/jsx-control-statements.latest.flow.js
```

Comment:
Using `npm` `3.10.10`, I have `./node_modules/jsx-control-statements/jsx-control-statements.flow.js` instead, resulting into the following message in `flow` logs:

> Skipping ./node_modules/jsx-control-statements/jsx-control-statements.latest.flow.js: No such file or directory

Reply:
Huh, guess I didn't `git add` the package.json changes. It should be fixed now if you run `npm install` again.

Reply:
(I'd also recommend doing `npm install -g npm@latest` if you can though, the latest versions of `npm` seem much less flaky/buggy for me.)